Actions-and-issues.md

```diff
@@ -7,17 +7,17 @@
 
 ## Introduction
 
-As a user plays through an exploration, we want to track their progress to identify any problems. If we do find problems with the exploration, we can raise those to the exploration creator. For example, if a user gets stuck in a cycle of cards or submits many incorrect answers to an interaction, the exploration could probably be improved.
+As a user plays through exploration, we want to track their progress to identify any problems. If we do find problems with the exploration, we can raise those to the exploration creator. For example, if a user gets stuck in a cycle of cards or submits many incorrect answers to an interaction, the exploration could probably be improved.
 
-We store the steps a user takes through an exploration as _actions_, and we create _issues_ to keep track of the problems that these actions indicate. Both actions and issues are defined as [[schemas|Schemas]].
+We store the steps a user takes through exploration as _actions_, and we create _issues_ to keep track of the problems that these actions indicate. Both actions and issues are defined as [[schemas|Schemas]].
 
 ## Actions
 
-There are three actions defined at [`extensions/actions/`](https://github.com/oppia/oppia/tree/develop/extensions/actions):
+There are three actions defined as [`extensions/actions/`](https://github.com/oppia/oppia/tree/develop/extensions/actions):
```

Comment:
I think `at` is correct here because the actions are defined inside the `actions` folder, not by the folder itself